src/server/game/Entities/AreaTrigger/AreaTriggerTemplate.h

```diff
@@ -164,6 +164,8 @@ class AreaTriggerMiscTemplate
         TimeToTarget        = 0;
         TimeToTargetScale   = 0;
 
+        ExtraScaleCurve     = { 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0000001f, 1.0f };
```

Comment:
Well the value still can be a float, so it's more logical to define it as float no ?